openassessment/xblock/openassessmentblock.py

```diff
@@ -422,6 +423,40 @@ def _(self):
         return i18nService.ugettext
 
     @property
+    def prompts(self):
+        """
+        Return the prompts.
+
+        Initially a block had a single prompt which was saved as a simple
+        string in the prompt field. Now prompts are saved as a serialized
+        list of dicts in the same field. If prompt field contains valid json,
+        parse and return it. Otherwise, assume it is a simple string prompt
+        and return it in a list of dict.
+
+        Returns:
+            list of dict
+        """
+        return create_prompts_list(self.prompt)
+
+    @prompts.setter
+    def prompts(self, value):
```

Comment:
Ha! I would have done that. I'll leave it up to you. Looking at the implementation I could see advantages to cleaning it up by having two fields, but also having that deprecated field is a little unfortunate.